### ISSUE TITLE
how it perfrom in  AVA dataset?

### ISSUE THREAD

**Issue by accountcwd** [wei-tim/YOWO#7]
thanks for open source code.

and i wonder know can YOWO run on the AVA dataset?

**Comment by okankop** [comment]
For AVA dataset, we need to formulate a new loss function since it is a multi-person multi-label dataset. We are currently working on it!

**Comment by accountcwd** [comment]
oh, that means your model‘s training with single-person label， but in infer it is able to output multi-person result?
I do not know if I have it wrong.

can i infer a large photo which contain many people？

**Comment by wei-tim** [comment]
@accountcwd 
In datasets UCF101-24 and J-HMDB-21, every single person has only one action label, but within an image, there can be several persons. As for AVA, it's not totally same, for each person might have two or more action labels.

You can use a large image containing several people, for each person there will be a single action prediction, like fencing in our examples.

**Comment by accountcwd** [comment]
> 
> 
> @accountcwd
> In datasets UCF101-24 and J-HMDB-21, every single person has only one action label, but within an image, there can be several persons. As for AVA, it's not totally same, for each person might have two or more action labels.
> 
> You can use a large image containing several people, for each person there will be a single action prediction, like fencing in our examples.

Got it ! Thanks for  detailed answer
@wei-tim  @okankop

**Comment by dagongji10** [comment]
> For AVA dataset, we need to formulate a new loss function since it is a multi-person multi-label dataset. We are currently working on it!

I have read your nice work for UCF101-24 dataset，and I wonder when will you finish the work for AVA ? Will you open source the AVA pretrained model ?
 If possible, could you give me an approximate date.

**Comment by okankop** [comment]
Hi @dagongji10, unfortunately we cannot give an approximate date now. Once we have YOWO working on AVA dataset, we would of course opensource pretrained models.

**Comment by okankop** [comment]
AVA results are coming soon! :fireworks: 

Stay tuned!

**Comment by irvingzhang0512** [comment]
> AVA results are coming soon! 🎆
> 
> Stay tuned!

Looking forward to your work. I'm planning to reproduce YOWO with [mmaction2](https://github.com/open-mmlab/mmaction2) someday.

**Comment by tomatowithpotato** [comment]
> AVA results are coming soon! 🎆
> 
> Stay tuned!

Will you launch yowo v2.0 in the future???????
looking forward to your work!!!

**Comment by okankop** [comment]
YOWO is extended for AVA dataset! Please check out the updated repo!

**Comment by okankop** [comment]
@tomatowithpotato yes I am working on YOWOv2, which will be faster and stronger!
